dropwizard-zipkin/src/main/java/smartthings/dw/asynchttpclient/TracingIOExceptionFilterProvider.java

```diff
@@ -0,0 +1,24 @@
+package smartthings.dw.asynchttpclient;
+
+import brave.http.HttpTracing;
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+import com.google.inject.Singleton;
+import org.asynchttpclient.filter.IOExceptionFilter;
+
+import static smartthings.brave.asynchttpclient.ClientTracing.TracingIOExceptionFilter;
+
+public class TracingIOExceptionFilterProvider implements Provider<IOExceptionFilter> {
```

Comment:
All of these special tracing filter providers should be necessary if we just use `ClientTracing.instrument(builder, tracer)` when the AHC instance is created.